openedx/core/djangoapps/content/block_structure/tests/test_models.py

```diff
@@ -79,6 +78,7 @@ def _verify_update_or_create_call(self, serialized_data, mock_log=None, expect_c
         return bsm
 
     @patch('openedx.core.djangoapps.content.block_structure.models.log')
+    @patch.dict(settings.BLOCK_STRUCTURES_SETTINGS, {'PRUNING_ACTIVE': False})
```

Comment:
This test needs to be explicitly excluded from pruning; it breaks the mock log assertions used in `_verify_update_or_create_call`